<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelDatasetHelper</class>
 <widget class="QWidget" name="ModelDatasetHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Model name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edModelName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Test model</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddRow">
       <property name="text">
        <string>Add row</string>
       </property>
       <property name="icon">
        <iconset resource="modeldataset.qrc">
         <normaloff>:/images/add_row.png</normaloff>:/images/add_row.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteRow">
       <property name="text">
        <string>Remove row</string>
       </property>
       <property name="icon">
        <iconset resource="modeldataset.qrc">
         <normaloff>:/images/remove_row.png</normaloff>:/images/remove_row.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddColumn">
       <property name="text">
        <string>Add column</string>
       </property>
       <property name="icon">
        <iconset resource="modeldataset.qrc">
         <normaloff>:/images/add_col.png</normaloff>:/images/add_col.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteColumn">
       <property name="text">
        <string>Remove column</string>
       </property>
       <property name="icon">
        <iconset resource="modeldataset.qrc">
         <normaloff>:/images/remove_col.png</normaloff>:/images/remove_col.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labType">
       <property name="text">
        <string>Column type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbCoulmnType">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="modeldataset.qrc"/>
 </resources>
 <connections/>
</ui>
